plugin.video.vrt.nu/resources/language/resource.language.nl_nl/strings.po

```diff
@@ -64,18 +64,38 @@ msgctxt "#32054"
 msgid "Woops something went wrong, check the log for more details"
 msgstr "Oeps, er ging iets mis, check de log voor meer informatie"
 
-msgctxt "#32091"
+msgctxt "#32080"
 msgid "A-Z"
 msgstr "A-Z"
 
-msgctxt "#32092"
+msgctxt "#32081"
+msgid "Alphabetically sorted list of TV programs"
+msgstr "Alphabetische lijst van TV programma's"
```

Comment:
Just a spelling remark:
In Dutch, tv is always written in lowercase and with a hyphen in compound words:
tv-programma's, tv-kanalen,...
https://onzetaal.nl/taaladvies/afkorting-in-samenstelling/
https://vrttaal.net/taaladvies-spelling/tv

Reply:
Thanks ! Fixed.